src/List.mo

```diff
@@ -25,10 +51,26 @@ module {
   };
 
   /// Construct a list by pre-pending a value.
-  /// This function is similar to a `list.cons(item)` function.
+  ///
+  /// Example:
+  /// ```motoko include=initialize
+  /// List.push<Nat>(0, null) // => ?(0, null);
+  /// ```
+  ///
+  /// Runtime: O(1)
+  ///
+  /// Space: O(1)
   public func push<T>(x : T, l : List<T>) : List<T> = ?(x, l);
```

Comment:
```suggestion
  public func push<T>(x : T, list : List<T>) : List<T> = ?(x, list);
```

Reply:
If you apply that logic consistently, then I guess we should have
```
 public func push<Elem>(elem: Elem, list : List<Elem>) : List<Elem> = ?(elem, list);
```
But I'm not sure that's actually better, and possibly worse, for legibility. Which is probably why the guidelines where encouraging short names.